src/components/website-management/website-form-components/customization/BannerCustomization.js

```diff
@@ -0,0 +1,59 @@
+import {Button, Grid, TextField} from "@mui/material";
+import React, {useState} from "react";
+import CustomizationTabs from "./CustomizationTabs";
+import BannerTemplate from "./customization-components/BannerTemplate";
+import CookieBanner from "./customization-components/CookieBanner";
+import {Box} from "@mui/system";
+
+
+
+    const BannerCustomization = (props)=>{
+
+        const {websiteId} = props;
+
+        console.log("this is website ID"+websiteId);
+
+
+        const [component,setComponent]= useState(<BannerTemplate/>);
+        const [value, setValue] = useState(0);
+
+
+        const handleNextClick = () => {
+            setComponent(<CookieBanner/>);
+            setValue(1);
+        };
+
+        const handleBackClick = () => {
+            setComponent(<BannerTemplate/>);
+            setValue(0);
+        };
+
+    return (
+        <Grid className={"customization"} container direction="row" justifyContent="space-evenly" alignItems="flex-start">
```

Comment:
Let's list the props for better readability. 

```suggestion
        <Grid 
            className="customization" 
            container 
            direction="row" 
            justifyContent="space-evenly" 
            alignItems="flex-start"
         >
```